<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>rForm</class>
 <widget class="QWidget" name="rForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Resultat - Sorteret p√• Kunder</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="cBoxLab" >
         <property name="minimumSize" >
          <size>
           <width>55</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>Kunde nr:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="knr" >
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="send_lst" >
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text" >
          <string>Mail Mig Listen</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>3</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label" >
             <property name="minimumSize" >
              <size>
               <width>55</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>55</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text" >
              <string>Firma/Navn</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labName" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2" >
             <property name="minimumSize" >
              <size>
               <width>55</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>55</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text" >
              <string>Att</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labAtt" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3" >
             <property name="minimumSize" >
              <size>
               <width>55</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>55</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text" >
              <string>Adresse</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labAddr" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4" >
             <property name="minimumSize" >
              <size>
               <width>55</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>55</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text" >
              <string>Post &amp; By</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labZipCity" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="exportXLS" >
           <property name="text" >
            <string>Exporter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeDlg" >
           <property name="text" >
            <string>Luk</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="lst" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
